evals/utils/api_utils.py

```diff
@@ -7,28 +7,28 @@
 
 import backoff
 import openai
+from openai import OpenAI
 
 EVALS_THREAD_TIMEOUT = float(os.environ.get("EVALS_THREAD_TIMEOUT", "40"))
 
 
 @backoff.on_exception(
     wait_gen=backoff.expo,
     exception=(
-        openai.error.ServiceUnavailableError,
-        openai.error.APIError,
-        openai.error.RateLimitError,
-        openai.error.APIConnectionError,
-        openai.error.Timeout,
+        openai.APIError,
+        openai.RateLimitError,
+        openai.APIConnectionError,
+        openai.Timeout,
```

Comment:
could we have someone who's familiar with how the new errors map to the old ones review this part? i think some 4xx shouldn't be retried (e.g. 401), and APIError is raised on both 4xx and 5xx (?)

Reply:
APIError, RateLimitError, and and APIConnectionErrors all map 1:1 here.

I think Timeout should actually be replaced with `APITimeoutError`

`ServiceUnavailableError` no longer exists, but i'll replace with `InternalServerError` as mentioned above.

Reply:
I've removed APIError as that's a grandparent exception to pretty much all errors from the API. All the others seem reasonable to keep